app/keepers/keepers.go

```diff
@@ -259,7 +254,7 @@ func (ak *AppKeepers) InitKeepers(
 	// set memory store keys
 	// NOTE: The testingkey is just mounted for testing purposes. Actual applications should
 	// not include this key.
-	ak.memKeys = storetypes.NewMemoryStoreKeys(capabilitytypes.MemStoreKey, "testingkey")
+	ak.memKeys = storetypes.NewMemoryStoreKeys("testingkey")
```

Comment:
https://github.com/babylonlabs-io/babylon/pull/989#discussion_r2123476898
Lets remove it than